wildfly/connectors/cassandra/connector-cassandra/pom.xml

```diff
@@ -34,6 +34,12 @@
             <groupId>org.teiid.connectors</groupId>
             <artifactId>translator-cassandra</artifactId>
         </dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.teiid.connectors</groupId>
+            <artifactId>cassandra-api</artifactId>
+            <version>14.0.0-SNAPSHOT</version>
```

Comment:
Remove the version here, we should not directly reference the version like that

Reply:
when i remove version, the maven declares the dependency to be unknown. I tried a few things to work this out but couldn't do. How to do this?

Reply:
It needs to go into parent pom under <DepencyManagement> section.

Reply:
May be this one I think https://github.com/teiid/teiid/blob/master/connectors/cassandra/pom.xml

Reply:
i think you are telling this for the above comment about addition of cassandra-api module.
What is to be done about the version thing?

Reply:
You need to add with the version here https://github.com/teiid/teiid/blob/master/pom.xml#L1635 then leave above as is without version.